### ISSUE TITLE
[FEATURE] Events Mobile - filter UI improvement

### ISSUE THREAD

**Issue by amandamartin-dev** [EddieHubCommunity/LinkFree#2902]
### Description

Now that there are 4 options to filter events, they are beginning to take alot of space especially on mobile.  This ticket is to modify the filter for mobile to a dropdown, smaller buttons, or another idea (open to discussion)

Current UI
![events page mobile](https://user-images.githubusercontent.com/97615019/211222719-363f697e-d776-49da-9128-6898f3bccba4.jpg)

**Comment by Aadarsh805** [comment]
@amandamartin-dev how about using the same tab format as what is being used in the user profile section like this 
![Group 5 (2)](https://user-images.githubusercontent.com/95094057/211274941-36fa0a5a-f398-4db2-9109-b7dcef11bc2e.png)

**Comment by amandamartin-dev** [comment]
@Aadarsh805 This is nice! One question, how will folks see the definitions for each filter item?  Should we put that in another location on mobile or is it obvious you think with these?

**Comment by Aadarsh805** [comment]
> @Aadarsh805 This is nice! One question, how will folks see the definitions for each filter item? Should we put that in another location on mobile or is it obvious you think with these?


I think we should have those definitions

#### One option would be to have a subheading that will be visible inside the tabs after clicking:
![Group 6](https://user-images.githubusercontent.com/95094057/211329890-98f4e1f5-4943-463f-8e62-89353d13d019.png)

#### Another option would be tooltips on hover:
![Screenshot from 2023-01-09 19-50-58](https://user-images.githubusercontent.com/95094057/211329982-b8231eae-00e3-4c9d-bc6d-5dd4a5be62f1.png)

**Comment by amandamartin-dev** [comment]
@Aadarsh805 oh, that's a good idea! Do you want me to assign you to work on this issue?

**Comment by Aadarsh805** [comment]
> @Aadarsh805 oh, that's a good idea! Do you want me to assign you to work on this issue?

Yep sure assign me, which one would I be implementing option 1 or 2?

**Comment by amandamartin-dev** [comment]
@Aadarsh805 I think option 1 bc option 2 won't work for mobile, right? and done! assigned you

**Comment by Aadarsh805** [comment]
> @Aadarsh805 I think option 1 bc option 2 won't work for mobile, right? and done! assigned you

Oh yeah, the hover won't work on mobile, thanks for assigning

**Comment by eddiejaoude** [comment]
Tab view for mobile changes to a drop down, I suggest we do something similar to the profile page with tabs for bigger screens and dropdown for mobile

**Comment by amandamartin-dev** [comment]
Ah yes, good point! Thanks Eddie

**Comment by Olatunji-seun23** [comment]
Perhaps I can up with a better UI design for this as a UI/UX designer

**Comment by Olatunji-seun23** [comment]
Okay then. so you will advice we just leave it instead cos of the stress of changing it on all platforms?

**Comment by Aadarsh805** [comment]
> Okay then. so you will advice we just leave it instead cos of the stress of changing it on all platforms?

Nah I didn't mean it like it'd be stressful changing it everywhere, I meant either we change it everywhere or not at all because it will disturb the UI consistency no
